firefox-ios/Client/Extensions/String+Extension.swift

```diff
@@ -74,4 +74,13 @@ extension String {
         }
         return boldString
     }
+
+    func attributedString(with style: [NSAttributedString.Key: Any]? = nil,
```

Comment:
i'd refactor the parameters name so they are more descriptive when the method is used.  The method could be named `attributedText` like the others.
I'd add a bit of documentation as well.

This is an example, if u have better naming go for it.
```swift
func attributedText(attributes: [NSAttributedString.Key: Any]? = nil, textToHighlight: String, 
// would set this as required otherwise what is the point of highlighting the text
highlightedTextAttributes: [NSAttributedString.Key: Any]) -> NSAttributedString
```

Reply:
Yes, I will add comments and update function name to attributedText.